README.md

```diff
@@ -125,7 +137,20 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+  #One Genie to One Lamp
+  #sample schema:
+  CREATE TABLE lamps (
+      id Serial PRIMARY KEY,
+      wishes_remaining INTEGER
+);
+  CREATE TABLE genies (
+      id Serial PRIMARY KEY,
+      name TEXT
+)
+  Lamp.new(wishes_remaining: 3 )
+  Genie.new(name:"Genie")
+#incomplete
```

Comment:
Remember, because this question assumes that a one-to-many relationship has been set up between Lamp and Genie, you can write something like this to established an association between two instances...

```rb
my_lamp = Lamp.new( wishes_remaining: 3 )
my_genie = Genie.new( name: "Genie" )
genie.lamp = my_lamp
```